R/fit.R

```diff
@@ -205,7 +214,8 @@ refit_multiple_optimizers <- function(fit,
 #'   optimizer_args = list(method = "L-BFGS-B")
 #' )
 mmrm_control <- function(n_cores = 1L,
-                         method = c("Satterthwaite", "Kenward-Roger", "Kenward-Roger-Linear"),
+                         method = c("Satterthwaite", "Kenward-Roger"),
+                         cov = NULL,
```

Comment:
hm so how would the robust sandwich be specified? And I thought that does not allow Satterthwaite or KR to be combined with it?

Reply:
it is specified in `cov` argument;  cov, covariance method; method, denominator degree of freedom method.
use `cov = "Empirical"` will give the sandwich estimator, just like 
![image](https://user-images.githubusercontent.com/36079400/213066359-a4b19430-f878-4c77-802e-a20337b9f309.png)
use `cov = "Empirical-Jackknife" will give CR3
![image](https://user-images.githubusercontent.com/36079400/213066541-35640a4c-27eb-4375-9652-a44fc0a13a98.png)


KR is not allowed to combine with Empirical/Empirical-Jackknife, but Satterthwaite is allowed; updated a little bit in the fit.R

Reply:
ah ok that is good to know! because I thought I remembered you saying that in SAS manual you cannot combine Satterthwaite d.f. and robust sandwich. But seems that is just a SAS restriction but not really based on the methods, right?

Reply:
Yes it is SAS restriction to only use "Contain" "Between-within" or "Residual" degree of freedom when the covariance is empirical. Although, Satterthwaite degree of freedom is developed based on the asymptotic covariance, the paper still uses this degree on empirical covariance for comparison.  see Bell, Robert M., and Daniel F. McCaffrey. "Bias reduction in standard errors for linear regression with multi-stage samples." Survey Methodology 28.2 (2002): 169-182.

![image](https://user-images.githubusercontent.com/36079400/213193442-e740ac98-5e78-4d7f-bbce-32d16efcf06e.png)

But I should further check if there the Satt d.f. changes if the covariance is robust/jackknife

Reply:
ok

Reply:
still reading the paper; the Sattherthwaite degree of freedom for Jackknife and empirical method are complicated